Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,15 +1,82 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+namespace BootCamp.Chapter1
 {
     public static class ArrayOperations
     {
+        // *************private helpers**********
         /// <summary>
+        /// Test that Array is neither null or empty.
+        /// </summary>
+        /// <returns>True if array can be manipulated by standard functions.</returns>
+        /// 
+        private static bool ValidateArrayForManipulation(int[] array)
+        {
+            return ((array != null) && (array.Length != 0));
+        }
+
+        /// <summary>
+        /// Makes the array one index item shorter.
+        /// </summary>
+        /// <returns>A new array with (original length - 1)</returns>
+        /// 
+        private static int[] MakeAShorterArray(int[] array)
+        {
+            return new int[array.Length - 1];
+        }
+
+        /// <summary>
+        /// Makes a new array with one index with an int
+        /// </summary>
+        /// <returns>A new array with a single index with value {number}>
+        /// 
+        private static int[] AddOneNumberToNullArray(int number)
+        {
+            return new[] { number };
+        }
+
+         /// <summary>
+        /// Makes the array one index item longer.
+        /// </summary>
+        /// <returns>A new array with (original length+ 1)</returns>
+        private static int[] MakeALongerArray(int[] array)
+        {
+            return new int[array.Length + 1];
+        }
```

Comment:
Same goes for those two functions. However:
It's very nice that you put commentary in!